examples/demo-cwl-project/agc-project.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: Demo
```

Comment:
Can you use DemoCwl or similar so that the deployed artifacts have naming that is easier to follow in testing?

Reply:
I will change this to be unique.